operators/index.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: ASSEMBLY
+[id="operators-overview"]
+= Operators overview
+include::modules/common-attributes.adoc[]
+:context: operators-overview
+
+toc::[]
+
+[id="importance-of-operators"]
+
+include::modules/operators-overview.adoc[leveloffset=+1]
+
+With Operators, you an create applications to monitor the running services in the cluster.
+Operators are designed specifically for your applications. Operators implement and automate the common Day 1 operations such as installation and configuration as well as Day 2 operations such as auto-scaling up and down and backups. All these activities are in a piece of software running inside your cluster.
+To understand more about Operators, see xref:../operators/understanding/olm-what-operators-are.adoc#olm-what-operators-are[What are Operators?]
+
+[id="overview-developer-tasks"]
+== For developers
+
+As a developer, you can perform the following Operator tasks:
```

Comment:
I list items with correct order as follows:

- [Install Operator SDK CLI](https://deploy-preview-41681--osdocs.netlify.app/openshift-enterprise/latest/operators/operator_sdk/osdk-installing-cli.html#osdk-installing-cli).
- Create [Go-based Operators](https://deploy-preview-41681--osdocs.netlify.app/openshift-enterprise/latest/operators/operator_sdk/golang/osdk-golang-quickstart.html#osdk-golang-quickstart), [Ansible-based Operators](https://deploy-preview-41681--osdocs.netlify.app/openshift-enterprise/latest/operators/operator_sdk/ansible/osdk-ansible-quickstart.html#osdk-ansible-quickstart), and [Helm-based Operators](https://deploy-preview-41681--osdocs.netlify.app/openshift-enterprise/latest/operators/operator_sdk/helm/osdk-helm-quickstart.html#osdk-helm-quickstart).
- [Use Operator SDK to build,test, and deploy an Operator](https://deploy-preview-41681--osdocs.netlify.app/openshift-enterprise/latest/operators/operator_sdk/osdk-about.html#osdk-about).
- [Install and subscribe an Operator to your namespace](https://deploy-preview-41681--osdocs.netlify.app/openshift-enterprise/latest/operators/user/olm-installing-operators-in-namespace.html#olm-installing-operators-in-namespace).
- [Create an application from an installed Operator through the web console](https://deploy-preview-41681--osdocs.netlify.app/openshift-enterprise/latest/operators/user/olm-creating-apps-from-installed-operators.html#olm-creating-apps-from-installed-operators).